From fc8b9ac29c03a7603a35f393713d1441046a6ccc Mon Sep 17 00:00:00 2001
From: Hans Rosenfeld <hans.rosenfeld@joyent.com>
Date: Thu, 18 Jan 2018 12:32:13 +0000
Subject: [PATCH] OS-6678 need bhyve modules in name_to_major Reviewed by:
 Jerry Jelinek <jerry.jelinek@joyent.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Patrick Mooney
 <patrick.mooney@joyent.com>

---
 overlay/generic/etc/name_to_major | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/overlay/generic/etc/name_to_major b/overlay/generic/etc/name_to_major
index 7f49def4..cfab8bff 100644
--- a/overlay/generic/etc/name_to_major
+++ b/overlay/generic/etc/name_to_major
@@ -298,3 +298,6 @@ iwn 301
 xhci 302
 qede 303
 buildstamp 304
+vmm 305
+viona 306
+ppt 307
-- 
2.21.0

